dbms/src/Common/RadixSort.h

```diff
@@ -247,14 +359,30 @@ struct RadixSort
 
         allocator.deallocate(swap_buffer, size * sizeof(Element));
     }
+
+    /* Most significant digit radix sort
+     * Usually slower than LSD and is not stable, but allows partial sorting
+     * Based on https://github.com/voutcn/kxsort
```

Comment:
You have partially copied some implementation details, but it requires to mention copyright and license.